client/src/main.rs

```diff
@@ -623,6 +624,7 @@ impl ClientState {
 			self.metrics
 				.record(MetricValue::DHTPutSuccess(success_rate));
 			self.metrics.record(MetricValue::DHTPutDuration(time_stat));
+			db.put(DhtPutSuccessKey, success_rate);
```

Comment:
We're using DB writes for every handled put request.
This will become a large overhead, just to be able to read it in the CI test.

And again, it should be under the build feature flag.